src/instances/arr/index.js

```diff
@@ -51,40 +51,43 @@ const Arr = (() => {
   };
 
   // :: (b -> a -> b) -> b -> [a] -> [b]
-  const scanl = f => a => match({
```

Comment:
Is this stuff just whitespace changes due to prettier?

Reply:
Yeah, I didn't have prettier set up when I added those originally.